charts/oauth/values.yaml

```diff
@@ -44,6 +44,8 @@ dex:
     scheme: "https"
     # if set to "non-existent", no Ingress resource will be created
     class: "nginx"
+    # list of ingress provider specific annotations
```

Comment:
Please describe that this is a map and it supports some sort of templating, cause from reading the code I am not quite sure what exactly you have in mind and an example would help a lot.